demos/capi/README.md

```diff
@@ -0,0 +1,89 @@
+# C and C++ API inference demo (C/C++)
+
+This demo creates a new docker image openvino/model_server-capi with two example applications that are using c or c++ ovms API.
+
+## Build ovms_shared.so
+To build the capi docker image, you must first build the ovms.tar.gz package with the libovms_shared.so library and ovms.h header. 
+run `make` command in ovms git main directory.
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+make
+```
+
+## Prepare demo image
+Next enter the directory with the example and build the demo docker with all dependencies and examples that will be named `openvino/model_server-capi`.
+The example image also contains dummy model and config.json required for the applications.
+```bash
+cd demos/capi
```

Comment:
What do you think about renaming the demo to something more explainatory to meet current convention? like c_api_minimal_app

```
ls demos
age_gender_recognition   face_blur                  image_classification_using_tf_model  person_vehicle_bike_detection          single_face_analysis_pipeline
benchmark                face_detection             model_ensemble                       README.md                              speech_recognition_with_kaldi_model
bert_question_answering  horizontal_text_detection  multi_faces_analysis_pipeline        real_time_stream_analysis              using_onnx_model
common                   image_classification       optical_character_recognition        segmentation_using_paddlepaddle_model  vehicle_analysis_pipeline
```

Reply:
Ok